chain-test/src/test/scala/org/bitcoins/chain/util/ChainUnitTest.scala

```diff
@@ -384,32 +277,38 @@ trait ChainUnitTest
     genesisHeaderF.map(_ => (chainHandler, zmqSubscriber))
   }
 
-  def createBitcoindChainHandler(): Future[BitcoindChainHandler] = {
+  def createBitcoindChainHandlerViaZmq(): Future[BitcoindChainHandlerViaZmq] = {
     composeBuildersAndWrap(createBitcoind,
                            createChainHandlerWithBitcoindZmq,
-                           BitcoindChainHandler.apply)()
+      BitcoindChainHandlerViaZmq.apply)()
   }
 
-  def destroyBitcoindChainHandler(
-      bitcoindChainHandler: BitcoindChainHandler): Future[Unit] = {
+  def destroyBitcoindChainHandlerViaZmq(
+      bitcoindChainHandler: BitcoindChainHandlerViaZmq): Future[Unit] = {
     val stopBitcoindF =
       BitcoindRpcTestUtil.stopServer(bitcoindChainHandler.bitcoindRpc)
     val dropTableF = destroyHeaderTable()
 
     bitcoindChainHandler.zmqSubscriber.stop
-
     stopBitcoindF.flatMap(_ => dropTableF)
   }
 
-  //BitcoindChainHandler => Future[Assertion]
-  def withBitcoindZmqChainHandler(test: OneArgAsyncTest)(
+  /**
+    * Creates a [[BitcoindRpcClient bitcoind]] that is linked to our [[ChainHandler bitcoin-s chain handler]]
```

Comment:
Nit: there's a difference between a bitcoind instance and a bitcoind RPC client instance